R/indentation_linter.R

```diff
@@ -50,11 +81,11 @@ indentation_linter <- function(indent = 2L, use_hybrid_indent = TRUE) {
     "number(",
     paste(
       c(
-        glue::glue("self::{paren_tokens}/following-sibling::{paren_tokens_right}/preceding-sibling::*[1]/@line2"),
-        glue::glue("self::*[{xp_and(paste0('not(self::', paren_tokens, ')'))}]/following-sibling::SYMBOL_FUNCTION_CALL/
+        glue::glue("self::{paren_tokens_left}/following-sibling::{paren_tokens_right}/preceding-sibling::*[1]/@line2"),
+        glue::glue("self::*[{xp_and(paste0('not(self::', paren_tokens_left, ')'))}]/following-sibling::SYMBOL_FUNCTION_CALL/
```

Comment:
too wide now :)

maybe an opportunity to switch to the style I sort of settled on elsewhere, where `/` paths become hanging indents if they're too wide:

```
self::*[...]
  /following-sibling::SYMBOL_FUNCTION_CALL
  /parent::expr
  /following-sibling::expr[1]
  /@line2
```